arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -291,7 +291,8 @@
               "enum": [
                 "Swarm",
                 "DCOS",
-                "Custom"
+                "Custom",
+                "Kubernetes"
```

Comment:
Yeah, that's true (although we're in that situation right now, the portal does use the Kubernetes type, so this is actually a fix, rather than a breaking change)

If you'd rather I rev'd the version to 11-01-2016 or something I can do that, but I actually think that it will be worse for users.